Code/Tools/AssetProcessor/native/unittests/RCcontrollerUnitTests.cpp

```diff
@@ -130,6 +130,7 @@ void RCcontrollerUnitTests::PrepareRCController()
 
 void RCcontrollerUnitTests::RunRCControllerTests()
 {
+    static constexpr int MaxProcessingWaitTimeMs = 60 * 1000; // Wait up to 1 minute.  Give a generous amount of time to allow for slow CPUs
```

Comment:
Three total minutes is exceedingly long. Do the BlockUntil calls exit early on completion?

I'm okay with this taking an entire second per call, but for me that's the edge of sanity for a "unit" scope test.

A better approach would be to not integrate with the entire QApplication, and mock out larger swathes of functionality around each function.

Reply:
Yeah they exit early, aside from the one EXPECT_FALSE which I made an exception for (kept it at 500ms).  This is just a maximum wait time, if the time is exceeded, the test will be marked as failed.

These calls are needed here to pump the qt event queue since these are Q_OBJECTS that communicate with slots and signals, so I don't think mocking would help much here.  Even so, it'd just bring us back to the situation of looping until this condition variable is true while waiting for the component to do the work we've queued.